fake-server/index.js

```diff
@@ -0,0 +1,31 @@
+const express = require("express")
+const bodyParser = require('body-parser');
+const fakeUsers = require("./fake-users.json");
+const app = express();
+const port = 3000
+
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+
+app.post("/", (req, res) => {
+    const { username, password } = req.body || {};
+    var policy = "";
+    if(!username || !password) {
+        res.status(400).send("Bad request");
+    } else {
+        let user = fakeUsers[username];
+        if(!user) {
+            res.status(403).send("Login failed");
+            console.log("Login failed for unknown user", username);
+        } else {

```

Comment:
This should be `else if { ...` etc.

Reply:
ok I've broken this out into functios instead. Hopefully it's enough.